src/app/page.tsx

```diff
@@ -33,6 +36,9 @@ export default function HomePage() {
           {homePageContent.linkButton.text}
         </NHSNotifyButton>
       </div>
+      <div className='nhsuk-grid-column-one-third'>
+        <Personalisation />
```

Comment:
Lets not add this onto the root page. (Assume this was just for testing).

Reply:
Yeah, mistake, taken off